chap7.tex

```diff
@@ -159,7 +159,7 @@ \subsection{Match Recognition}
   \item Linear support vector machine (SVM) with L2 regularization (squared
     Euclidean norm)
   \item SVM with radial basis function kernel (SVM-RBF), L2 regularization
-  \item Perceptron (P)
```

Comment:
[Perceptron](https://en.wikipedia.org/wiki/Perceptron) is the correct one.
I am going to accept this and correct it later